src/js/cf-expandables.js

```diff
@@ -5,110 +5,168 @@
  * A public domain work of the Consumer Financial Protection Bureau
  */
 
-(function( $ ) {
+( function( $ ) {
+
+  'use strict';
+
+  var _extend = $.extend;
+
+  var Expandable = {
+
+    defaultProperties: {
+      isInAccordion: false,
+      isExpanded:    false,
+      expandedClass: 'expandable__expanded'
+    },
+
+    // Elements that will be cached.
+    ui: {
+      cueOpen:  '.expandable_cue-open',
+      cueClose: '.expandable_cue-close',
+      content:  '.expandable_content',
+      target:   '.expandable_target'
+    },
+
+    create: function create( options ) {
+      var properties = _extend( true, {}, Expandable.defaultProperties, options );
+
+      // Using $.extend to create an instance of the Expandable.
+      // Extending dom elements is generally bad and should be
+      // avoided. `this` is referring to the dom element.
+      _extend( this, Expandable, properties );
```

Comment:
You are better off using the jQuery data element ( which isn't the same as the HTML5 data attribute btw ) but I didn't want to introduce breaking changes.